bananas_api/web_routes/config.py

```diff
@@ -18,6 +18,7 @@
 
 BRANCHES = {
     Branch.OFFICIAL_OPENTTD: "Official OpenTTD",
+    Branch.JGRPP: "JGRPP",
```

Comment:
Github repo and forums topic write the ful name as "JGR's Patchpack" or "JGR's Patch Pack".
Maybe @JGRennison has a preferred spelling.

Reply:
I've got no objections to JGRPP.
Increasingly the source doesn't look like a pack of patches anyway.